recipes-openxt/qtbase/qtbase_git.bbappend

```diff
@@ -0,0 +1,38 @@
+PACKAGECONFIG = " \
+    release \
+    libpng \
+    udev \
+    dbus \
+    tools \
+    libs \
+"
+
+
+PACKAGECONFIG[libinput] = ""
+
+QT_CONFIG_FLAGS += " \
+    -dbus \
+    -no-xcb \
+"
+
+PACKAGES_remove = "${PN}-examples-dev"
+
+PACKAGECONFIG_CONFARGS += " \
+    -no-accessibility \
+    -no-cups \
+    -no-qml-debug \
+    -no-sql-mysql \
+    -no-sql-sqlite \
+    -no-openssl \
+    -no-eglfs \
+    -no-opengl \
+    -no-xcb \
+    -no-icu \
+    -release \
+    -no-iconv \
+    -nomake examples \
+    -nomake tests \
+"
+
+DEPENDS += "libpng"
+PARALLEL_MAKE = ""
```

Comment:
I bet it is necessary, but still... :'(

Reply:
Yea @jean-edouard ran into a bunch of build problems on the build server with PARALLEL_MAKE on.  I did not see any issues on my local box.  Made it hard to debug.  Decided to punt it down the road.

Reply:
That only reproduces when using many CPU cores, like on the AIS builder.
See builds $6421 to $6423.